Solutions/Box/Data Connectors/azuredeploy_Box_API_FunctionApp.json

```diff
@@ -128,7 +128,7 @@
                 "alwaysOn": true,
                 "reserved": true,
                 "siteConfig": {
-                    "linuxFxVersion": "python|3.6"
+                    "linuxFxVersion": "python|3.8"
```

Comment:
In the connector documentation we should explicitly mention that in case of any change to the function app box custom app should be re-authorize.

Reply:
@rushriva 
Box custom app does not need to be re-authorized after function app changes. The issue was that Box custom app should be re-authorized after changes in Box custom app itself.